content/src/fieldConfig/env-questionnaire.json

```diff
@@ -0,0 +1,80 @@
+{
+  "envReqQuestionsPage": {
+    "generic": {
+      "title": "Tell Us About Your Business",
+      "question": "Which of the following are true?",
+      "buttonText": "See My Results",
+      "errorText": "Select a response below.",
+      "footerText": "To learn more about ${mediaAreaText} permits, visit ${mediaAreaLink}."
+    },
+    "waste": {
+      "mediaAreaText": "waste",
+      "mediaAreaLink": "[NJDEP’s page on Sustainable Waste Management](https://www.nj.gov/dep/dshw/)",
+      "questionnaireOptions": {
+        "hazardousMedicalWaste": "My business will generate `hazardous waste|hazardous-waste` or `medical waste|medical-waste`.",
+        "compostWaste": "My business will `compost waste|compost-waste` on site.",
+        "treatProcessWaste": "My business will treat or process waste.",
+        "constructionDebris": "My business will process `construction/demolition debris|construction-and-demo-debris` and recyclable materials.",
+        "noWaste": "None of the above apply."
+      }
+    },
+    "land": {
+      "mediaAreaText": "land management",
+      "mediaAreaLink": "[NJDEP’s page on the Watershed and Land Management Program](https://dep.nj.gov/wlm/)",
+      "questionnaireOptions": {
+        "takeOverExistingBiz": "My business will take over an existing business or a site where a business used to be.",
+        "propertyAssessment": "My business will carry out `property assessment|property-assessment` work at the site.",
+        "constructionActivities": "My business will carry out construction activities on a site affecting 0.25 or more acres of land.",
+        "siteImprovementWasteLands": "My business will carry out site improvement in or near wetlands or flood hazard areas. This includes activities such as cutting down trees.",
+        "noLand": "None of the above apply."
+      }
+    }
+  },
+  "envReqResultsPage": {
+    "title": "Actions to Take",
+    "editInfo": "Your answers were successfully completed!",
+    "editText": "Edit.",
+    "summary": {
+      "ptOne": "Based on your answers, your business may be legally required to have a Department of Environmental Protection’s (DEP)",
+      "ptTwo": "license or permit. Find out your business' requirements below.",
```

Comment:
dust - personally not a huge fan or abbreviations like Req and pt.

Reply:
I don't have a strong preference for one or another so I spelled them out!